rules/windows/process_creation/proc_creation_win_net_default_accounts_manipulation.yml

```diff
@@ -0,0 +1,56 @@
+title: Suspicious Manipulation Of Default Accounts
+id: 5b768e71-86f2-4879-b448-81061cbae951
+status: experimental
+description: Detects suspicious manipulations of default accounts such as 'administrator' and 'guest'. For example 'enable' or 'disable' accounts or change the password...etc
+author: Nasreddine Bencherchali
+references:
+    - https://www.trellix.com/en-sg/about/newsroom/stories/threat-labs/lockergoga-ransomware-family-used-in-targeted-attacks.html
+    - https://redacted.com/blog/bianlian-ransomware-gang-gives-it-a-go/
+date: 2022/09/01
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        Image|endswith:
+            - '\net.exe'
+            - '\net1.exe'
+    selection_user_option:
+        CommandLine|contains: ' user '
+    selection_username:
+        CommandLine|contains:
+            # Note: We need to write the full account name for cases starting with 'admin' to avoid lookups only with the user flag
+            - ' Järjestelmänvalvoja ' # Finish
+            - ' Rendszergazda ' # Hungarian
+            - ' Администратор ' # Russian
+            - ' Administrateur ' # French
+            - ' Administrador ' # Portuguese (Brazil + Portugal) + Spanish
+            - ' Administratör ' # Swedish
+            - ' Administrator ' # English
+            - ' guest '
+            # The cases below are for when an attacker requests the net command via 'cmd /c....'
+            # First in double quotes
+            - ' "Järjestelmänvalvoja" ' # Finish
+            - ' "Rendszergazda" ' # Hungarian
+            - ' "Администратор" ' # Russian
+            - ' "Administrateur" ' # French
+            - ' "Administrador" ' # Portuguese (Brazil + Portugal) + Spanish
+            - ' "Administratör" ' # Swedish
+            - ' "Administrator" ' # English
+            - ' "guest" '
+            # Second in single quotes
+            - " 'Järjestelmänvalvoja' " # Finish
+            - " 'Rendszergazda' " # Hungarian
+            - " 'Администратор' " # Russian
+            - " 'Administrateur' " # French
+            - " 'Administrador' " # Portuguese (Brazil + Portugal) + Spanish
+            - " 'Administratör' " # Swedish
+            - " 'Administrator' " # English
+            - " 'guest' "
+    condition: all of selection_*
```

Comment:
Could we exclude cases in which accounts get disabled? E.g. 
```
filter:
    CommandLine|contains: '/active no'
```
I could imagine that legitimate scripts disable the guest user account. 
```
net user guest /active no
```

Reply:
Yes, good catch.